.env.example

```
@@ -0,0 +1,4 @@
+DB_HOST=10.250.0.103
+DB_NAME=easyevent
+DB_USER=easyevent
+DB_PASS=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)